backend/api/src/Routes/routes.go

```diff
@@ -81,6 +86,17 @@ func authMiddleware(c *gin.Context) {
 	c.Set("UserID", id)
 }
 
+func adminMiddleware(c *gin.Context) {
+	id := c.Keys["UserID"].(uint)
+	var err error
+	var isAdmin bool
+	isAdmin, err = db.IsRole(gormDB, id, "admin")
```

Comment:
bör kunna ändras till shorthand assignment med `:=`
och deklarationen på raderna ovan kan tas bort.

```suggestion
	isAdmin, err := db.IsRole(gormDB, id, "admin")
```